scenario/multi-backend/assets/saio/phase1/etc/rc.local

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+#
+# rc.local
+#
+# This script is executed at the end of each multiuser runlevel.
+# Make sure that the script will "exit 0" on success or any other
+# value on error.
+#
+# In order to enable or disable this script just change the execution
+# bits.
+#
+# By default this script does nothing.
+
+mkdir -p /var/cache/swift /var/cache/swift2 /var/cache/swift3 /var/cache/swift4
+chown swift: /var/cache/swift*
```

Comment:
If the user is hardcoded here, we should not let someone else change the linux user here. I suggest we go with 'swift' everywhere and don"t let this as an option. Otherwise you need to make this file a template.